src/types/user.type.ts

```diff
@@ -1,29 +1,35 @@
 export interface UserData {
-    __caslSubjectType__ : 'UserData'
-    id: string;
-    email: string
-    displayName: string
-    photoURL: string
-    given_name: string,
-    family_name: string,
-    nickname: string,
-    name: string,
-    email_verified?: boolean
-    user_metadata?: any
-    staff?: boolean;
-    defaultAbteilung?: string;
-    roles: { [abteilungId: string]: role }
+  __caslSubjectType__: "UserData";
+  id: string;
+  email: string;
+  displayName: string;
+  photoURL: string;
+  given_name: string;
+  family_name: string;
+  nickname: string;
+  name: string;
+  email_verified?: boolean;
+  user_metadata?: any;
+  staff?: boolean;
+  defaultAbteilung?: string;
+  roles: { [abteilungId: string]: role };
 }
 
 export interface UserDataUpdate {
-    email: string
-    displayName: string
-    photoURL: string
-    given_name: string,
-    family_name: string,
-    nickname: string,
-    name: string,
-    defaultAbteilung?: string;
+  email: string;
+  displayName: string;
+  photoURL: string;
+  given_name: string;
+  family_name: string;
+  nickname: string;
+  name: string;
+  defaultAbteilung?: string;
 }
 
-export type role = 'pending' | 'guest' | 'member' | 'matchef' | 'admin';
\ No newline at end of file
+export type role = "pending" | "guest" | "member" | "matchef" | "admin";
+
+export interface AbteilungMember {
```

Comment:
Ich habe ein Typescript interface erstellt, damit alles auch getyped ist. Casl wird nur benötigt, wenn man recht überprüfen will. (https://casl.js.org/v6/en/guide/intro). Also z.B. `ability.can('read', 'AbteilungMember')`.

Das wird aber in diesem Beispiel nicht benötigt.